### ISSUE TITLE
Error in migration engine

### ISSUE THREAD

**Issue by srohimah** [prisma/prisma#20633]
Hi Prisma Team! The following command just crashed.

## Command

`db push`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | debian-openssl-1.1.x| 
| Node        | v16.10.0           | 
| Prisma CLI  | 4.6.1              | 
| Engine      | 694eea289a8462c80264df36757e4fdc129b1b32| 

## Error
```
Error: Error in migration engine.
Reason: [migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs:1019:22] We should only be setting a changed default if there was one on the previous schema and in the next with the same enum.

Please create an issue with your `schema.prisma` at
https://github.com/prisma/prisma/issues/new

```

**Comment by Jolg42** [comment]
Hi @srohimah 👋🏼 
It looks like this was fixed in 4.7.0, could you try upgrading to that version or a more recent one?

PR: https://github.com/prisma/prisma-engines/pull/3379

If this is not already fixed, we would need more information here to get to a reproduction.

**Comment by Jolg42** [comment]
@srohimah I'll close this as it should be fixed already, let us know if it's indeed the case or if we should reopen this.
